### ISSUE TITLE
Image profiles support

### ISSUE THREAD

**Issue by fanningert** [fanningert/kirbycms-extension-image#4]
Possible to define in the `config.php` some image profiles.

Example:

``` php
$img_profiles = array();
$img_profiles['template'] = {
  'width' => 200,
  'height' => 200,
  'resize' => 'crop',
  'grayscale' => true,
  ...
};
c::set('kirby.extension.imageext.profiles', $img_profiles);
c::set('kirby.extension.imageext.default_profile', 'template');
```

Usage of different profiles with the tag and overwrite some profile settings.

``` php
(imagext: picture.jpg profile: template blur: true)
```

**Comment by fanningert** [comment]
Profiles support is included in version 0.9. Only the default profile parameter is at the moment not implemented.

**Comment by fanningert** [comment]
Is now working
